hash/a7800.xml

```diff
@@ -2461,7 +2461,7 @@ almost nothing like the prototype.
 	<!-- remove them? -->
 
 	<software name="dkongxmps" cloneof="dkongxm" supported="no">
-		<description>Donkey Kong (PAL, Demo, XM enhanced, V1.2, for prosystem emulator)</description>
+		<description>Donkey Kong (PAL, demo, XM enhanced, V1.2, for prosystem emulator)</description>
```

Comment:
We could make ProSystem camel case here, as on the 7800 itself. I guess we should also capitalize Emulator then, because it seems this is referring to the specific emulator named ProSystem Emulator.

Reply:
That’s LlamaCase aka PascalCase aka JavaCase.  camelCase doesn’t start with a hump.

Reply:
Both are camel ;) :
upper camel case (initial uppercase letter, also known as Pascal case or bumpy case) and lower camel case (initial lowercase letter, also known as dromedary case)

Reply:
Camel, Llama, Dromedary... replaced the "prosystem emulator" by "ProSystem Emulator", before it turns to a zoo